### PR TITLE
feat: add Ad Inspector

### PR DESCRIPTION
**Pull request by wjaykim**
### Description

This PR adds [Ad Inspector](https://developers.google.com/ad-manager/mobile-ads-sdk/android/ad-inspector) functionality.

### Release Summary

Added AdInspector Module

### Checklist

- I read the [Contributor Guide](https://github.com/invertase/react-native-google-mobile-ads/blob/main/CONTRIBUTING.md)
  and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `__tests__e2e__`
  - [ ] `jest` tests added or updated in `__tests__`
- [x] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

Updated example app.

---

Think `react-native-google-mobile-ads` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

File: example/src/AdInspector.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import {Test, TestType, TestResult} from 'jet';
+import React from 'react';
+import {View, Button, StyleSheet} from 'react-native';
+import MobileAds from 'react-native-google-mobile-ads';
+
+export class AdInspectorTest implements Test {
+  getPath(): string {
+    return 'AdInspectorTest';
+  }
```
Comment (mikehardy) [inline example/src/AdInspector.tsx:9]: I hate to argue for "worse" code, but could you put this directly in the App.tsx? The reason is that the example app is built by a script, and it has to copy relevant files out while it rebuilds from react-native init, so it's easier to keep the example just as one file for copy out / copy in

That's why the file is huge and everything is in there all in one spot :-)

**Review by mikehardy — CHANGES_REQUESTED** [review]
This looks really neat! Might be worth a mention in the README?
And one thing to change about the example, but it's easy - this all looks great
The iOS CI fail was something unrelated, I restarted it

File: example/src/AdInspector.tsx (hunk 1 above)
Comment (wjaykim) [reply example/src/AdInspector.tsx:9]: Okay, I understand.

**Review by mikehardy — APPROVED** [review]
This is fantastic! I pulled it locally and ran it, easy to run in the example and the results are great. I think this will really help people, thanks for implementing it!

**Pull request merged**